scripts/Diffraction/isis_powder/polaris_routines/polaris_algs.py

```diff
@@ -83,7 +83,7 @@ def save_unsplined_vanadium(vanadium_ws, output_path):
 
 
 def generate_ts_pdf(run_number, focus_file_path, merge_banks=False, q_lims=None, cal_file_name=None,
-                    sample_details=None, output_binning=None, pdf_type="G(r)", freq_params=None):
+                    sample_details=None, delta_r=None, delta_q=None, pdf_type="G(r)", freq_params=None):
```

Comment:
Is this is a public or implementation API? 

Previously the code assumed

`isis_powder/polaris.py  # Public API methods go here`
`isis_powder/polaris_routines/*.py  # Private API methods go here`

If this is still true we should drop the message saying output_binning is changed in the release notes (or else users may worry about having to make changes to scripts).

Alternatively, if this is now a public API users are expected to call, we should possible look at some sort of deprecation warning so users know what to do with their scripts.

Reply:
However the output binning option was only added in this release and it is unlikly anyone is already using it.

Reply:
Ah so we haven't had a Mantid release with the output binning option?

Reply:
It was added in 5.0. if an incorect paramiter is given to the function it wil raise a warning right away and list the valid paramiters, so in this case it should be clear what is wrong.

Reply:
Okay, this is going to be better to get done sooner rather than later so scripts don't build up, so I agree 👍